packages/vow/src/tools.js

```diff
@@ -54,10 +54,17 @@ export const prepareBasicVowTools = (zone, powers = {}) => {
   /**
    * Vow-tolerant implementation of Promise.all.
    *
-   * @param {EVow<unknown>[]} maybeVows
+   * @param {unknown[]} maybeVows
    */
   const allVows = maybeVows => watchUtils.all(maybeVows);
 
+  /**
+   * Vow-tolerant implementation of Promise.allSettled.
```

Comment:
I think this needs more documentation and maybe a rename.

It works on promises, not just vows. There should be a test showing so,
```js
  const result = await when(
    allVowsSettled([vowA, vowB, vowC, Promise.reject(new Error('an error'))]),
  );
```

Please clarify that in the JSDoc. Also maybe call it `allSettled` since it isn't just vows? That is does support vows is somewhat implied by being in `prepareVowTools`.

Reply:
> maybe a rename

I agree this naming is more clear, but I followed the precedent set by `allVows` which also accepts any Passables. Should `allVows` be renamed to `all`?  cc @michaelfig 

> It works on promises, not just vows. There should be a test showing so

We have `allVows can accept passable data (PureData)` which tests other non-vow Passables but not Promises. Happy amend that test to include a Promise and demonstrate more specimen types in the `all(Vows)Settled` tests

Agree RE better docs, I felt a smidge of guilt writing `n/a` in the PR body section. I'll aim for a README section on `VowTools` in addition to doc strings

Reply:
I would prefer to see `vowAll` and `vowAllSettled`, to rhyme better with `Promise.all` and `Promise.allSettled`.

Reply:
> I would prefer to see `vowAll` and `vowAllSettled`, to rhyme better with `Promise.all` and `Promise.allSettled`.

But given that they'd typically be found in `vowTools`, `vowTools.all` and `vowTools.allSettled` would be acceptable.